firebase-auth/src/jsMain/kotlin/dev/gitlive/firebase/auth/externals/auth.kt

```diff
@@ -170,6 +170,14 @@ public external interface AuthResult {
     public val credential: AuthCredential?
     public val operationType: String?
     public val user: User?
+    public val additionalUserInfo: AdditionalUserInfo?
+}
+
+public external interface AdditionalUserInfo {
+    public val providerId: String?
+    public val username: String?
+    public val profile: Map<String, Any?>?
```

Comment:
is this not a map, it will be of type [Json](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.js/-json/)

Reply:
I used `dynamic` in the end